indico_jacow/client/PeerReviewManagersFileInput.jsx

```diff
@@ -0,0 +1,215 @@
+// This file is part of the JACoW plugin.
+// Copyright (C) 2021 - 2024 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+
+import uploadManagersFileURL from 'indico-url:plugin_jacow.peer_review_managers_import'
+
+import PropTypes from 'prop-types';
+import React, {useCallback, useState} from 'react';
+import {useDropzone} from 'react-dropzone';
+import {Button, Message, MessageHeader, MessageList, 
+    MessageItem, Icon, Loader, Segment, Dimmer} from 'semantic-ui-react';
+import {Field} from 'react-final-form';
+
+import {FinalField, handleSubmitError} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import {SingleFileArea} from 'indico/react/components/files/FileArea';
+import {FinalModalForm} from 'indico/react/forms/final-form';
+import {indicoAxios} from 'indico/utils/axios';
+
+import './PeerReviewManagerFileInput.module.scss';
+
+const PeerReviewManagersFileInput = ({
+    onFocus,
+    onBlur,
+    onChange,
+    validExtensions,
```

Comment:
No need to pass this as a prop since it's always going to be the same, so you can just hardcode it below.